src/core/InstrumentFunctions.cpp

```diff
@@ -479,11 +480,14 @@ void InstrumentFunctionArpeggio::processNote( NotePlayHandle * _n )
 			cur_arp_idx = (int)( range * ( (float) rand() / (float) RAND_MAX ) );
 		}
 
+		// Divide cur_arp_idx with wanted repeats. This method doesn't work with random though.
+		cur_arp_idx = (int)( cur_arp_idx / m_arpRepeatsModel.value() );
```

Comment:
I think the comment is a bit clearer, before it sounded like there was something wrong with this line for random directions.
```suggestion
		// Divide cur_arp_idx with wanted repeats. The repeat feature will not affect random notes.
		cur_arp_idx = (int)(cur_arp_idx / m_arpRepeatsModel.value());
```